cmd/posix-errors.go

```diff
@@ -58,8 +55,7 @@ func isSysErrNoSpace(err error) bool {
 // Input/output error
 func isSysErrIO(err error) bool {
 	if pathErr, ok := err.(*os.PathError); ok {
-		switch pathErr.Err {
-		case syscall.EIO:
+		if pathErr.Err == syscall.EIO {
```

Comment:
Further simplification is possible simply do `return pathErr.Err == syscall.EIO`